qiskit_ibm_runtime/ibm_backend.py

```diff
@@ -73,13 +79,61 @@ def __init__(
             configuration: Backend configuration.
             api_client: IBM Quantum client used to communicate with the server.
         """
-        super().__init__(configuration=configuration)
-
+        super().__init__(
+            name=configuration.backend_name,
+            online_date=configuration.online_date,
+            backend_version=configuration.backend_version,
+        )
         self._api_client = api_client
-
-        # Attributes used by caching functions.
+        self._configuration = configuration
         self._properties = None
+        self._qubit_properties = None
         self._defaults = None
+        self._target = None
+        self._max_circuits = configuration.max_experiments
+
+    def __getattr__(self, name: str) -> Any:
+        self._get_properties()
+        self._get_defaults()
+        self._convert_to_target()
+        try:
+            return super().__getattribute__(name)
+        except AttributeError:
+            pass
+        try:
+            return self._configuration.__getattribute__(name)
+        except AttributeError:
+            raise AttributeError(
+                "'{}' object has no attribute '{}'".format(
+                    self.__class__.__name__, name
+                )
+            )
```

Comment:
I did a bit on research myself (e.g. [[1]](https://stackoverflow.com/questions/3278077/difference-between-getattr-vs-getattribute)) but am still struggling to understand these lookup fallbacks. Can you elaborate a bit (or, ideally add a code comment that explains a little background here)?

Reply:
I've added doc string and comments.

Reply:
Thanks, that makes it much clearer.